src/components/common/interfaces.tsx

```diff
@@ -53,7 +53,11 @@ export interface foodItemProps {
 export interface foodcardDetailsProps {
   [key: string]: foodItemProps[];
 }
-
+export interface PaymentCardDetailsProps {
+  cardNumber: string;
```

Comment:
cardNumber should be number right ?

Reply:
It also contains letters "2345 XXXX XXXX 6574"

Reply:
it does not contain only numbers "2345 XXXX XXXX 6574"